core/src/HIP/Kokkos_HIP_KernelLaunch.hpp

```diff
@@ -170,6 +288,67 @@ struct HIPParallelLaunchKernelInvoker<DriverType, LaunchBounds,
   }
 };
 
+// HIPLaunchMechanism::GlobalMemory specialization
+template <typename DriverType, typename LaunchBounds>
```

Comment:
Please comment with the line ranges that are identical in `Kokkos_Cuda_KernelLaunch.hpp`.


Also, if we're going to copy/paste things like this, please don't make unnecessary stylistic changes like `template <class...>` to `template <typename...>`. This just makes it harder for someone to come along later with a diff tool and figure out what the salient differences are. (Again, I would argue that the fact that we're even discussing a reader having to use a diff tool is a **major** problem, but if I'm going to lose that argument, please at least don't make unnecessary stylistic changes that make it harder for the reader to even use diff).